lambdas/writeCollection/index.js

```diff
@@ -18,7 +18,12 @@ exports.handler = async function (event, context) {
     "type": "Collection",
     "label": {
       "none": [
-        "This is the label"
+        "Maktaba"
+      ]
+    },
+    "summary": {
+      "none": [
+        "Maktaba is a digital collection of African Arabic manuscripts, translations, and scholarly works, developed by Northwestern University's Institute for the Study of Islamic Thought in Africa (ISITA) and the University of Illinois at Urbana-Champaign's Center for African Studies (CAS). The project was initially supported by the National Endowment for the Humanities, with technical development led by Northwestern University Libraries' Development team."
```

Comment:
Let's use Rebecca's phrasing. I think it offers more context. 

"Maktaba is an open-access digital collection of translated and contextualized Arabic manuscripts from Muslim West Africa. A collaborative effort between Northwestern University and the University of Illinois at Urbana-Champaign, Maktaba is dedicated to making portions of the African manuscript collections from these universities’ libraries accessible to a wide variety of users for learning, teaching, and research."